### DISCUSSION TITLE
Which branch is stable right now? 2.3 or 2.4?

### DISCUSSION THREAD

**Discussion by ndanyluk** [supermerill/SuperSlicer#3194] [Q&A]
Hi, been using 2.3.57.12 for a long time - I see that the usual "this is the stable version" disclaimer hasn't been included in the last 2 releases tagged on the repo.  Is 2.4.x safe for everyday use or should I wait it out a bit more until more bugs can be fixed

**Comment by supermerill** [comment]
I'll push another batch of fixes into the 2.4 this week. I advise to at least wait it.

**Reply by ndanyluk** [reply]
> I'll push another batch of fixes into the 2.4 this week. I advise to at least wait it.

Thanks for the fixes! Is 2.4 considered stable now or is another release planned before it gets that designation?
